<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>user</string>
  </property>
  <widget class="QLineEdit" name="LE_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_password">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="placeholderText">
    <string>password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_createUser">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Create User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_hrId">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="placeholderText">
    <string>hr_id</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>HR Id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_fullName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Full name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Full name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>User type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_userType">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch ID</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_branch">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_userStatus">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
